app/controllers/parks_controller.rb

```diff
@@ -0,0 +1,57 @@
+class ParksController < ApplicationController
+
+  def index
+    if params[:search].present?
+      @parks = Park.near(params[:search], 50).page params[:page]
```

Comment:
https://en.wikipedia.org/wiki/Magic_number_(programming)
thoughts on extracting it into a constant? `SEARCH_RADIUS = 50`